site/templates/page/page-content.jet

```diff
@@ -4,8 +4,22 @@
       {{text | raw}}
 
       {{if plan != 0 }}
-      <div>
+      <div class="plan-buttons">
+        {{ if plan.PlanType == "free" }}
+          {{ if plan.HasExpiryDate() }}
+          <s72-before datetime="{{ plan.ExpiryDate.UnixMilli() }}">
+            <s72-user-anon>
+              <a class="btn btn-block s72-btn-purchase signup-link" href="{{i18n("plan_free_link_url")}}">{{ i18n("plan_free_link_text") }}</a>
```

Comment:
Given that `plan_free_link_url` isn't actually a translation (and using it without `routeToPath` will always send you to the default language page and trigger a redirect), I would prefer the path hardcoded into the template `href="{{routeToPath("/signup.html")}}"`.  If it is supposed to be configurable, it would make more sense as a config that as a translation hack.